sampleproject/components/greeting.py

```diff
@@ -11,7 +11,9 @@ def get(self, request, *args, **kwargs):
         context = {"name": request.GET.get("name", "")}
         return self.render_to_response(context, slots)
 
-    def get_context_data(self, name, *args, **kwargs) -> Dict[str, Any]:
+    def get_context_data(self, *args, **kwargs) -> Dict[str, Any]:
+        # NOTE: Because of MyPy in Python v3.6, arg `name` cannot be declared as separate variable
```

Comment:
In older Python versions (definitely 3.6, don't know about 3.7 and 3.8), MyPy complained when a subclassed method signature had extra arguments, even if followed by `*args`.

But this brings up a topic - Is it time to drop support for 3.6 and 3.7? After all, [they have reached end of life](https://devguide.python.org/versions/)

Reply:
I think it's a good idea to drop support eventually, but it might be worth keeping it for this PR because the required change is small enough.

I wonder if there's a way to estimate how many projects are using Python 3.6 or Python 3.7. I tried using some of the search features on GitHub but it didn't work.

Reply:
The policy we're trying to follow is to support the same versions of python that django supports. So all officially supported Django versions -> All supported Python versions that they support: https://docs.djangoproject.com/en/5.0/faq/install/#what-python-version-can-i-use-with-django

Open to discussing other policies (in separate issue!) :)